ccd-definition/AuthorisationCaseType.json

```diff
@@ -22,5 +22,11 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
```

Comment:
Should gatekeeper be CRU or R and same for judiciary, surely these should match?

Reply:
I think the user just needs to Read the case type. Doesn't update or create currently